hangman.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html>
+<head>
+  <title>HANGMAN!</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <h1>HANGMAN!</h1>
+  <div>
+  <img src="1.png">
+    <p id="p1"></p>
+    <p id="p2">Pick a letter!</p>
+    <input type="text">
+    <button onclick="hangman()">Guess</button>
+  <p id="p3">LETTERS USED :</p>
+  </div>
+  <script>
```

Comment:
It is not a bad practice to put all JS codes in script tag this time as JS code is only 80 lines long. In the future, try to create separate js file and link it to html document for better organizing practice.